internal/service/advancedcluster/model_advanced_cluster.go

```diff
@@ -275,6 +288,18 @@ func SchemaAdvancedConfig() *schema.Schema {
 					Type:     schema.TypeInt,
 					Optional: true,
 				},
+				"custom_openssl_cipher_config_tls12": {
+					Type:     schema.TypeSet,
+					Optional: true,
+					Elem: &schema.Schema{
+						Type: schema.TypeString,
+					},
+				},
+				"tls_cipher_config_mode": {
+					Type:     schema.TypeString,
+					Optional: true,
+					Computed: true,
```

Comment:
@maastha @AgustinBettati @EspenAlbert  I would try to avoid new Optional+Computed attributed because of TPF and follow "effective values" in [IPA 111](https://docs.devprod.prod.corp.mongodb.com/ipa/111).

For example having `tls_cipher_config_mode` as Optional, and `effective_ tls_cipher_config_mode` as Computed. WDYT?

Reply:
Effective values makes sense to me when it implemented on the API side. Custom logic in tf handling 2 separate attributes from a single API property seems to move away from our practices of staying close to the API.
What challenges does TPF face with this Optional/Computed case, compared to SDK v2?

Reply:
>is this being returned by the API even when not defined? Just want to confirm

@AgustinBettati yes. API returns always returns `DEFAULT` even if not configured so this is okay for being Optional & Computed. 

@lantoli Adding to @AgustinBettati 's comment. I'm not sure I understand why we want to use effective values in this case?

Reply:
I thought C+O attributes inside blocks (like this one in advanced_configuration) were the main cause to have breaking changes in TPF, isn't that correct?